aws-cpp-sdk-core/source/external/cjson/cJSON.cpp

```diff
@@ -120,7 +120,7 @@ CJSON_AS4CPP_PUBLIC(double) cJSON_AS4CPP_GetNumberValue(const cJSON * const item
 CJSON_AS4CPP_PUBLIC(const char*) cJSON_AS4CPP_Version(void)
 {
     static char version[15];
-    sprintf(version, "%i.%i.%i", CJSON_AS4CPP_VERSION_MAJOR, CJSON_AS4CPP_VERSION_MINOR, CJSON_AS4CPP_VERSION_PATCH);
+    snprintf(version, sizeof(version), "%i.%i.%i", CJSON_AS4CPP_VERSION_MAJOR, CJSON_AS4CPP_VERSION_MINOR, CJSON_AS4CPP_VERSION_PATCH);
```

Comment:
are we sure we want to be laying changes on top of this copy/pasted CJSON implementation?

if we take an update, won't these changes just get wiped away?

Reply:
since this is external code, it might be simpler to just disable the warnings from the top of the file, or in the CMakeLists.txt somehow ... something isolated with a high chance of being preserved if someone copy/pastes an updated cJSON.cpp into the repo

... discuss with the CPP team how they want to handle it

Reply:
Since we are already C99, since we have already modified version of cJSON, we will unblock existing mac builds (i.e. I'm for merging this change).
And deal with external dependencies later.